spawn.md

```diff
@@ -0,0 +1,84 @@
+
+# Electron Spawning Process
+
+## Two processes
+* Main  
+  launching browser, you can use any nodejs api.   
+  it's full control
+  can use native api
+
+* Renderer  
+  inside browser v8 runtime.
+  it's limited control.  
+  cannot use native directly    
+
+## Yarn setup
+1. Install yarn in nodejs  
+    sudo npm -g install yarn
+2. Create electron app  
+    yarn create electron-app my-app  
+    cd my-app  
+3. Run the program  
+    yarn start  
+4. Package the program  
+     yarn make  
+
+## Spawn Process    
+1. Add package     
+    yarn add execa    
+2. Open source   
+    cd src  
+   code index.js   
+3. Spawn the process   
+```
+const { app, BrowserWindow } = require("electron");
+const { exec } = require("child_process");
+const execa = require("execa");
+
+if (require("electron-squirrel-startup")) {
+  app.quit();
+}
+
+function run_program() {
+  (async () => {
+    const { stdout } = await execa("/usr/local/bin/client-rpc", [
+      "--network-id",
+      "ab"
```

Comment:
Better make program location, storage location and network id configurable via environment variable with a default value.

Reply:
yes
also for macosx, we should give absolute path to execa

Reply:
And also the Tendermint RPC (HTTP + WS) parameters should be configurable, as one may not run the full node on its machine or configure it on different ports.
One could possibly bundle tendermint binary as well and launch `tendermint lite` as a proxy, but we had some issues with it before

Reply:
i will add two options
--tendermint-url
--websocket-url